lib/test/test-7/test-7.test.js

```diff
@@ -207,25 +207,14 @@ describe('XML is partially filled, class values are valid', () => {
               score: 4,
               name: 'V1',
               vulnerabilityId: 1
-            },
-            {
-              score: null,
-              name: '',
-              vulnerabilityId: 2
-            },
+            }
           ],
           attackPathName: 'V1 AND ',
           attackPathScore: 4,
         },
         {
           riskAttackPathId: 2,
-          vulnerabilityRef: [
-            {
-              score: null,
-              name: '',
-              vulnerabilityId: 1
-            },
-          ],
+          vulnerabilityRef: [],
```

Comment:
The scenario is Risk ID 2 has two attack paths with 2 vulnerabilities, 1 and 2. Vuln 2 does not have a name so the attack path 2 is empty. When exporting this to sra format, the vulnerabilityRef is an empty array.